src/main/java/com/radixdlt/crypto/ECDSASignatures.java

```diff
@@ -0,0 +1,30 @@
+package com.radixdlt.crypto;
+
+import com.radixdlt.serialization.SerializerId2;
+
+import java.util.Map;
+
+@SerializerId2("crypto.ecdsa_signatures")
+public class ECDSASignatures extends SignaturesImpl<ECDSASignature> {
```

Comment:
Why do we need this as a class? 
ECDSASignatures could just be created on the call site using something like SignaturesImpl.for(ECDSASignature.class)?

Reply:
It is the container providing the `@SerializerId2("crypto.ecdsa_signatures")`

Reply:
I thought the signature class was supposed to be self-contained?

Reply:
It also provides convenience, simpler constructors. But how can we put Serializer ID into `SignaturesImpl`, and make it be dynamic? Not possible with annotations I guess?